src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -184,6 +276,29 @@ private void InternalSetPosition(XYZ fi)
             TransactionManager.Instance.TransactionTaskDone();
         }
 
+        private void InternalSetPosition(Line pos)
+        {
+            TransactionManager.Instance.EnsureInTransaction(Document);
+
+            //var lp = InternalFamilyInstance.get_Parameter(BuiltInParameter.CURVE_IS_REFERENCE_LINE).Set(pos.Reference.ElementId);
+            var lp = InternalFamilyInstance.Location as LocationCurve;
+            lp.Curve = pos;
+
+            TransactionManager.Instance.TransactionTaskDone();
+        }
+
+        private void InternalSetFace(Reference face)
```

Comment:
Should we call this method InternalSetHostFace(), how do you validate that the input reference belongs to face?